### DISCUSSION TITLE
Instance broke after attempt to move some specific files

### DISCUSSION THREAD

**Discussion by SamuXzX** [nextcloud/all-in-one#2942] [Questions] [answered]
### Steps to reproduce
1. Selected some files from the Android App (https://cloud.samuelezappala.it/s/GFN8q972ksYHtCn)
2. Selected option to move them (https://cloud.samuelezappala.it/s/zaww3tBSjBe4WAX)
3. Selected destination folder (https://cloud.samuelezappala.it/s/tDHteAmE4DaRF7G)

### Expected behavior 
Files moving to the destination folder.

### Actual behavior 
The view got back to the folder the file were in, I received a toast message for every file I tried to move, saying something along the lines of "Wrong setting of the server".
For some time the Android App continued successfully syncing with the server (or at least I didn't receive any alert or message about an unreachable server).

After some hours Davx5 on my phone tried to sync the calendars and for every calendar I received an 504 error.
Log of Davx5: [https://cloud.samuelezappala.it/s/fwGX6DWwQCgTCTm](https://cloud.samuelezappala.it/s/fwGX6DWwQCgTCTm)

Then I tried reaching a note in the Android Note app and it stopped showing me an error.
Log of Android Note app: [https://cloud.samuelezappala.it/s/bFpJig6SFsABmmG](https://cloud.samuelezappala.it/s/bFpJig6SFsABmmG)

After restarting the VPS, everything started working again. I succeeded in moving the files via the browser interface with my pc.

### Host OS 
Ubuntu 22.04.2 LTS (GNU/Linux 5.15.0-60-generic x86_64)

#### Nextcloud AIO version 
6.2.1

#### Current channel 
latest

#### Other valuable info 
I entered in Portainer, I noticed that the nextcloud-database container was tagged as "Unhealty".

Database container log: [https://cloud.samuelezappala.it/s/zbY8Pw4wTCe8ca9](https://cloud.samuelezappala.it/s/aSH7sAj8ZPxSEHL)

Nextcloud container log: [https://cloud.samuelezappala.it/s/aSH7sAj8ZPxSEHL](https://cloud.samuelezappala.it/s/aSH7sAj8ZPxSEHL)

Nextcloud log from Nextcloud UI instead of the Nextcloud container: [https://cloud.samuelezappala.it/s/RLaPiNGBjJTX9cb](https://cloud.samuelezappala.it/s/RLaPiNGBjJTX9cb)

Apache log: [https://cloud.samuelezappala.it/s/ajLLJbsEaqW4fz4](https://cloud.samuelezappala.it/s/ajLLJbsEaqW4fz4)

**Comment by SamuXzX** [comment]
Trying to move those same 6 files from the new folder to another folder (this time, the root) caused the same issue. The toast message says in italian: "Configurazione non corretta del server", which translates as "Wrong server configuration". 
[https://cloud.samuelezappala.it/s/2F5HS8ETM4QS4o2](https://cloud.samuelezappala.it/s/2F5HS8ETM4QS4o2)

Similar database container log: [https://cloud.samuelezappala.it/s/TsWSHpYBTFAm6rp](https://cloud.samuelezappala.it/s/TsWSHpYBTFAm6rp)

I'm restarting the VPS and attaching the files here:
[Pinguini Tattici Nucleari - Fede.odt](https://github.com/nextcloud/all-in-one/files/12009293/Pinguini.Tattici.Nucleari.-.Fede.odt)
[Pinguini Tattici Nucleari - Giulia.odt](https://github.com/nextcloud/all-in-one/files/12009294/Pinguini.Tattici.Nucleari.-.Giulia.odt)
[Pinguini Tattici Nucleari - Hold on.docx](https://github.com/nextcloud/all-in-one/files/12009295/Pinguini.Tattici.Nucleari.-.Hold.on.docx)
[Pinguini Tattici Nucleari - Nonono.odt](https://github.com/nextcloud/all-in-one/files/12009296/Pinguini.Tattici.Nucleari.-.Nonono.odt)
[Pinguini Tattici Nucleari - Ricordi.docx](https://github.com/nextcloud/all-in-one/files/12009297/Pinguini.Tattici.Nucleari.-.Ricordi.docx)
[Pinguini Tattici Nucleari - Scooby Doo.odt](https://github.com/nextcloud/all-in-one/files/12009298/Pinguini.Tattici.Nucleari.-.Scooby.Doo.odt)

I tried with other files and all went well:
[Branduardi - Confessioni di un malandrino.docx](https://github.com/nextcloud/all-in-one/files/12009307/Branduardi.-.Confessioni.di.un.malandrino.docx)
[Damien Rice -_.docx](https://github.com/nextcloud/all-in-one/files/12009308/Damien.Rice.-_.docx)
[Guccini - Cristoforo Colombo.docx](https://github.com/nextcloud/all-in-one/files/12009309/Guccini.-.Cristoforo.Colombo.docx)
[Guccini - Vedi cara.docx](https://github.com/nextcloud/all-in-one/files/12009310/Guccini.-.Vedi.cara.docx)
[Guccini - Vorrei.docx](https://github.com/nextcloud/all-in-one/files/12009311/Guccini.-.Vorrei.docx)
[Jovanotti - Baciami ancora.docx](https://github.com/nextcloud/all-in-one/files/12009312/Jovanotti.-.Baciami.ancora.docx)

The only difference I see is that the first six comprehend .odt files.
Moving only the two .docx files of such group caused no issues.

**Comment by szaimen** [comment]
Hi, how many files have been in the folder that you tried to move?

**Comment by SamuXzX** [comment]
I didn't move any folder; the folder icon you see is probably an error of the Android App, which sometimes shows me files with a folder icon.

**Comment by szaimen** [comment]
Okay sorry, how many files did you try to move?

**Comment by SamuXzX** [comment]
6: 4 odt and 2 docx.
You can see it in the first link.

**Comment by szaimen** [comment]
Thanks! I will try to reproduce with the docs that you shared!

**Comment by szaimen** [comment, marked as answer]
So I tried to reproduce the issue with the files that you sent but I could not unfortunately. Moving to discussions for now thus.

**Comment by SamuXzX** [comment]
I'm pretty sure I would be able to reproduce it again. I posted all the logs, maybe we can investigate further?

**Reply by szaimen** [reply]
You could try to disable some nextcloud apps and check if that helps...

**Reply by SamuXzX** [reply]
Do you have suggestions on what to start with?

**Reply by emelbardis** [reply]
Try disabling Music. It’s been causing a current bug in NC to display errors on file moves/deletes

**Reply by SamuXzX** [reply]
I don't have it installed, although I did some time ago.
